app/managers/network_manager.ts

```diff
@@ -60,7 +60,7 @@ class NetworkManager {
         sessionConfiguration: {
             allowsCellularAccess: true,
             waitsForConnectivity: false,
-            httpMaximumConnectionsPerHost: 10,
+            httpMaximumConnectionsPerHost: 100,
```

Comment:
this change increases the amount of concurrent requests that can be perform for each server, specially significant on Android